package.json

```diff
@@ -38,6 +38,7 @@
     "pusher-js": "^8.4.0-rc2",
     "randexp": "^0.5.3",
     "react": "^18.2.0",
+    "react-avatar": "^5.0.3",
```

Comment:
We definitely need to kick out this library. It wasn't updated in years and it will definitely block our path to React 19.

As replacement we can use ui-avatars.com, but we should be careful here as well.

@turbo124 should we use 3rd party website to generate nice avatars for us (due to privacy concerns).

Just to be clear `react-avatar` isn't fully offline either.